Source/CryptRaider/AI/Classic/Tasks/BTTask_SetIsPursuing.cpp

```diff
@@ -0,0 +1,30 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "BTTask_SetIsPursuing.h"
+
+#include "AIController.h"
+#include "BehaviorTree/BlackboardComponent.h"
+
+UBTTask_SetIsPursuing::UBTTask_SetIsPursuing()
+{
+	NodeName = "Set Is Pursuing";
+
+	BlackboardKey.AddBoolFilter(this,
```

Comment:
I think we could leave this new line, but do it right under `this` as it done in
`Source/CryptRaider/AI/Classic/Tasks/BTTask_SearchInRadius.cpp`